scripts/exgfs_wave_init.sh

```diff
@@ -38,27 +38,21 @@ for grd in ${WAVECUR_FID} ${WAVEICE_FID} ${WAVEWND_FID} ${waveuoutpGRD} ${waveGR
 done
 
 for grdID in "${grdALL[@]}"; do
-  if [[ -f "${COMOUT_WAVE_PREP}/${RUN}.wave.t${cyc}z.mod_def.${grdID}.bin" ]]; then
-    echo "INFO: Mod def file for ${grdID} found in ${COMOUT_WAVE_PREP}. copying ...."
-    cpreq "${COMOUT_WAVE_PREP}/${RUN}.wave.t${cyc}z.mod_def.${grdID}.bin" "mod_def.${grdID}"
-
+  echo "INFO: Mod def file for ${grdID} not found in ${COMOUT_WAVE_PREP}. Setting up to generate ..."
```

Comment:
We didn't check to see if the `mod_def` file is present, so the comment should instead say:
```suggestion
  echo "INFO: Setting up to generate mod_def file for ${grdID}"
```

Reply:
Great catch!